src/github/nag.rs

```diff
@@ -973,16 +1086,43 @@ impl<'a> RfcBotComment<'a> {
                 msg.push_str("). :bell:");
 
                 if !added_label {
-                    msg.push_str("\n\n*psst @");
-                    msg.push_str(&author.login);
-                    msg.push_str(", I wasn't able to add the `final-comment-period` label,");
-                    msg.push_str(" please do so.*");
+                    Self::couldnt_add_label(&mut msg, author, LABEL_FCP);
                 }
 
                 msg
             }
 
-            CommentType::FcpWeekPassed => "The final comment period is now complete.".to_string(),
+            CommentType::FcpWeekPassed {
+                author,
+                added_label,
+                status_comment_id,
+                disposition
+            } => {
+                let mut msg = String::new();
+                msg.push_str("The final comment period, with a disposition to **");
+                msg.push_str(disposition.repr());
+                msg.push_str("**, as per the [review above](");
+                Self::add_comment_url(issue, &mut msg, status_comment_id);
+                msg.push_str("), is now **complete**.");
+
+                if is_rfc_repo(issue) {
+                    match disposition {
+                        FcpDisposition::Merge => {}
+                        FcpDisposition::Close => {
+                            msg.push_str("\n\nBy the power vested in me by Rust, I hereby close this RFC.");
```

Comment:
Note that this won't be an accurate comment to post until rfcbot is given more permissions on the `rust-lang` org. Can we make this conditional on whether actually closing the issue succeeded? That might require a slight change to order of operations.